go.mod

```diff
@@ -32,6 +32,8 @@ require (
 	github.com/libp2p/go-libp2p-kad-dht v0.12.2
 	github.com/libp2p/go-libp2p-kbucket v0.4.7
 	github.com/libp2p/go-libp2p-pubsub v0.4.1
+	github.com/libp2p/go-libp2p-transport-upgrader v0.4.2
```

Comment:
`go mod tidy` and maybe commit go sum also?

Reply:
done, nothing changed.